File: BAEKJOON/Node/10000/10700/10797.js
const fs = require("fs")
const filepath = process.platform === "linux" ? "/dev/stdin" : "./input.txt"
const input = fs.readFileSync(filepath).toString().trim().split("\n")

const N = parseInt(input[0])
let ans = 0
const seq = input[1].split(" ").map(x => parseInt(x))
for (var k in seq) {
  if (seq[k] === N) {
    ans++
  }
}
console.log(ans)

File: BAEKJOON/Python/10000/10700/10797.py
"""
Title : 10부제
Link : https://www.acmicpc.net/problem/10797
"""

from sys import stdin

input = stdin.readline


if __name__ == "__main__":
    N = int(input())
    seq = list(map(int, input().split()))
    print(seq.count(N))
